Packs/FeedInfobloxThreatIntelligence/Integrations/FeedInfobloxThreatIntelligence/FeedInfobloxThreatIntelligence.yml

```
@@ -0,0 +1,400 @@
+category: Data Enrichment & Threat Intelligence
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: InfobloxThreatIntelligenceFeed
+  version: -1
+configuration:
+- section: Connect
+  display: ""
```

The `api_key` parameter is not configured correctly. Please use the `display` field for the label and remove the `displaypassword` and `hiddenusername` fields, as they are not applicable for a `type: 9` API key.

```suggestion
  display: Service API Key
  name: api_key
  type: 9
  required: true
  additionalinfo: 'Service API key for Infoblox TIDE API authentication'
```